nucypher/characters/base.py

```diff
@@ -279,21 +287,16 @@ def verify_from(self,
                     message_kit: Union[UmbralMessageKit, bytes],
                     signature: Signature = None,
                     decrypt=False,
-                    delegator_verifying_key: UmbralPublicKey = None,
                     ) -> tuple:
```

Comment:
Hmm - `cleartext` is a `tuple`? `Tuple[type, type]` will help me read this a bit quicker, in this case.

Reply:
I think that `tuple` is just stale code. I changed this return type to `bytes`.